docker/get-package-name.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
This can probably safely be deleted as well, it shouldn't be referenced anywhere. I'll make a note to delete this in the other repos as well. This is the deprecated way to auto-detect image names.